src/main/java/step3/domain/CarName.java

```diff
@@ -0,0 +1,26 @@
+package step3.domain;
+
+import step3.exception.WrongGameSettingException;
+
+public class CarName {
+
+    private static final int CAR_NAME_MAX_LENGTH = 5;
+
+    private final String carName;
+
+    public CarName(String carName) {
+        validateCarName(carName);
+        this.carName = carName;
+    }
+
+    private void validateCarName(String carName) {
+        if (carName.length() > CAR_NAME_MAX_LENGTH) {
+            throw new WrongGameSettingException("자동차 이름은 5자를 초과할 수 없습니다.");
+        }
+    }
+
+    public String getString() {
+        return this.carName;
+    }
```

Comment:
```suggestion
    public String value() {
        return this.carName;
    }
```
이런 메소드 명이 더 어울리지 않을까요?

Reply:
답변감사합니다! 원시타입포장객체에는 동일한 메서드를 사용해 통일감을 주도록 하겠습니다.